spec/routes/directory.spec.js

```diff
@@ -49,7 +49,7 @@ describe('directory route', function () {
         });
     });
 
-    it('returns multiple employees', function (done) {
+    it.only('returns multiple employees', function (done) {
```

Comment:
Please disregard this commentary. I've just noticed you remove `only` from this example. Thanks!